shared/src/commonMain/kotlin/io.newm.shared/internal/implementations/SignupUseCaseImpl.kt

```diff
@@ -10,9 +10,9 @@ import kotlin.coroutines.cancellation.CancellationException
 internal class SignupUseCaseImpl(private val repository: LogInRepository) : SignupUseCase {
 
     @Throws(KMMException::class, CancellationException::class)
-    override suspend fun requestEmailConfirmationCode(email: String, humanVerificationCode: String) {
+    override suspend fun requestEmailConfirmationCode(email: String, humanVerificationCode: String, mustExists: Boolean) {
```

Comment:
@martyu When calling request `requestEmailConfirmationCode` from the forget password user flow we need to set `mustExists = true`. However, when calling  `requestEmailConfirmationCode` from account creation`mustExists = false`